cookbooks/aws-parallelcluster-environment/resources/efa/efa_alinux2.rb

```diff
@@ -24,6 +24,10 @@ def conflicting_packages
   end
 
   def prerequisites
-    %w(environment-modules libibverbs-utils librdmacm-utils)
+    if aws_region.start_with?("us-iso")
+      %w(libibverbs-utils librdmacm-utils)
+    else
+      %w(environment-modules libibverbs-utils librdmacm-utils)
+    end
```

Comment:
Why is environment-modules removed?

Reply:
same reasoning as the other comment I left